### PR TITLE
React Data Fetching: Demo

### PR DESCRIPTION
**Pull request by djfarly**
### 🔎 Review

Review locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/react-data-fetching_demo/sessions/react-data-fetching/demo-start -i
```

### ✨ Copyable Markdown for Curriculum

````md
Use this demo locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/react-data-fetching/demo-start -i
```
````

### 🔎 Review

Review locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/react-data-fetching_demo/sessions/react-data-fetching/demo-end -i
```

### ✨ Copyable Markdown for Curriculum

````md
You can check out the final version of this demo locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/react-data-fetching/demo-end -i
```
````

### CODE REVIEW

File: sessions/react-data-fetching/demo-end/pages/_app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+const fetcher = (url) => fetch(url).then((r) => r.json());
+
+export default function App({ Component, pageProps }) {
+  return (
+    <SWRConfig value={{ fetcher }}>
+      <GlobalStyle />
+      <Component {...pageProps} />
+    </SWRConfig>
+  );
```
[the remark this answers is not in the document]
Comment (djfarly) [reply sessions/react-data-fetching/demo-end/pages/_app.js:12]: yea thats better - i was just lazy

File: sessions/react-data-fetching/demo-start/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# React Data Fetching: Jokes
```
Comment (PReichetanz) [inline sessions/react-data-fetching/demo-start/README.md:1]: As we have already talked about, adapt the README.md files to the correct content of a demo.

**Review by PReichetanz — CHANGES_REQUESTED** [review]
The demo works nicely as expected!

We just have to fix the readme.md files, then we are done.

File: sessions/react-data-fetching/demo-start/README.md (hunk 2 above)
Comment (PReichetanz) [reply sessions/react-data-fetching/demo-start/README.md]: Update: adapted the readme files myself.

**Review by PReichetanz — APPROVED** [review]
I adapted the readme files myself, everything should be fine now.

@djfarly Just check the readme files again, that's it.

File: sessions/react-data-fetching/demo-end/components/Joke/index.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  const { isFunny } = info;
+
+  return (
+    <>
+      <small>ID: {id}</small>
+      <h1>
+        {data.joke}{" "}
+        <span
+          role="img"
+          aria-label={isFunny ? "A Funny joke" : "Not a funny joke"}
```
Comment (djfarly) [inline sessions/react-data-fetching/demo-end/components/Joke/index.js:55]:
```suggestion
          aria-label={isFunny ? "A laughing face" : "An unamused face"}
```

**Pull request merged**